js/Genoverse.js

```diff
@@ -133,7 +134,7 @@ var Genoverse = Base.extend({
 
       if (browser.loadedPlugins[plugin] || $('script[src="' + js + '"]').length) {
         getCSS();
-      } else {
+      } else if (!runningInModule) {
```

Comment:
You still need to add `done()`

Reply:
Could be it works like that. But I think done resolves `deffered`. Could be wrong

Reply:
I checked - it still works correctly in storybook, it doesn't try to re-load the plugins

Reply:
👍